src/NeXusDataset.cpp

```diff
@@ -5,20 +5,34 @@
  *  \brief Implement datasets used by the ADC file writing module.
  */
 
-#include "Datasets.h"
+#include "NeXusDataset.h"
 
 namespace NeXusDataset {
 RawValue::RawValue(hdf5::node::Group parent, Mode CMode, int ChunkSize)
     : ExtensibleDataset<std::uint16_t>(parent, "raw_value", CMode, ChunkSize) {}
 
 Time::Time(hdf5::node::Group parent, Mode CMode, int ChunkSize)
-    : ExtensibleDataset<std::uint64_t>(parent, "time", CMode, ChunkSize) {}
+    : ExtensibleDataset<std::uint64_t>(parent, "time", CMode, ChunkSize) {
+  if (Mode::Create == CMode) {
+    auto StartAttr = ExtensibleDataset::attributes.create<std::string>("start");
+    StartAttr.write("1970-01-01T00:00:00Z");
```

Comment:
See the [documentation for NXlog](http://download.nexusformat.org/doc/html/classes/base_classes/NXlog.html?highlight=nxlog) (We are storing the time as time since UNIX Epoch).